py/desitarget/mock/build.py

```diff
@@ -147,8 +147,11 @@ def read_mock(params, log=None, target_name='', seed=None, healpixels=None,
     zmin_lya = params.get('zmin_lya')
     zmax_qso = params.get('zmax_qso')
     use_simqso = params.get('use_simqso', True)
+    sqmodel=params.get('sqmodel','default')
     balprob = params.get('balprob', 0.0)
     add_dla = params.get('add_dla', False)
+    add_metals=params.get('add_metals', False)
+    add_lyb=params.get('add_lyb', False)
 
```

Comment:
Just a very minor stylistic quibble/request to add spaces between the `=` here, and throughout (basically to match the current style).